services/runner/runner.go

```diff
@@ -25,6 +25,10 @@ var (
 	ErrBadProxyAPI = fmt.Errorf("proxy api must be specified as http(s) when scan api is websocket")
 )
 
+const (
+	ContainerTerminateTimeout = time.Second * 30
```

Comment:
Can we make this 5m? Supervisor kills many containers. And ungraceful exit in supervisor means tearing down running bots during auto-updates.